sdk/formrecognizer/azure-ai-formrecognizer/README.md

```diff
@@ -85,14 +85,30 @@ client = FormRecognizerClient(endpoint, credential)
 ## Key concepts
 
 ### FormRecognizerClient
-A `FormRecognizerClient` is the Form Recognizer interface to use for analyzing receipts, recognizing content from
-forms, and recognizing data from forms using custom trained models. It provides different methods
+A `FormRecognizerClient` is the Form Recognizer interface to use for recognizing data from forms using custom trained models,
+recognizing content and layout from forms, and analyzing receipts. It provides different methods
 based on inputs from a URL and inputs from a stream.
```

Comment:
Might be worth saying something like different methods for providing the input document as a file stream or Uri path to the document.